tests_basic/test_fieldparams.py

```diff
@@ -0,0 +1,245 @@
+"""Tests for the flatsym module of pylinac."""
```

Comment:
Definitely appreciate you putting together tests. This is what separates the men from the boys =)